Ref/SignalGen/SignalGen.fpp

```diff
@@ -35,6 +35,18 @@ module Ref {
     # Special ports
     # ----------------------------------------------------------------------
 
+    @ Data product get port
+    product get port productGetOut
+
+    @ Data product request port
+    product request port productRequestOut
```

Comment:
Why is there bothj get and request on this component and one is left in TODO state?